docs/sdk/src/guides/enable_elastic_scaling_mvp.rs

```diff
@@ -89,17 +82,63 @@
 #![doc = docify::embed!("../../cumulus/polkadot-omni-node/lib/src/nodes/aura.rs", slot_based_colator_import)]
 //!
 //! 2. In `start_consensus()`
-//!     - Remove the `overseer_handle` param (also remove the
+//!     - Remove the `overseer_handle` and `relay_chain_slot_duration` params (also remove the
 //!     `OverseerHandle` type import if it’s not used elsewhere).
-//!     - Rename `AuraParams` to `SlotBasedParams`, remove the `overseer_handle` field and add a
-//!     `slot_drift` field with a   value of `Duration::from_secs(1)`.
-//!     - Replace the single future returned by `aura::run` with the two futures returned by it and
-//!     spawn them as separate tasks:
+//!     - Rename `AuraParams` to `SlotBasedParams`, remove the `overseer_handle` and
+//!     `relay_chain_slot_duration` fields and add a `slot_drift` field with a value of
+//!     `Duration::from_secs(1)`. Also add a `spawner` field initialized to
+//!     `task_manager.spawn_handle()`.
+//!     - Replace the `aura::run` with the `slot_based::run` call and remove the explicit task
+//!       spawn:
 #![doc = docify::embed!("../../cumulus/polkadot-omni-node/lib/src/nodes/aura.rs", launch_slot_based_collator)]
 //!
-//! 3. In `start_parachain_node()` remove the `overseer_handle` param passed to `start_consensus`.
-//!
-//! ### Phase 2 - Activate fixed factor scaling in the runtime
+//! 3. In `start_parachain_node()` remove the `overseer_handle` and `relay_chain_slot_duration`
+//!    params passed to `start_consensus`.
+//!
+//! ### Phase 2 - Configure core selection policy in the parachain runtime
+//!
+//! With the addition of [RFC-103](https://polkadot-fellows.github.io/RFCs/approved/0103-introduce-core-index-commitment.html),
+//! the parachain runtime has the responsibility of selecting which of the assigned cores to build
+//! on. It does so by implementing the `SelectCore` trait.
+#![doc = docify::embed!("../../cumulus/pallets/parachain-system/src/lib.rs", SelectCore)]
+//!
+//! For the vast majority of use cases though, you will not need to implement a custom core
+//! selector. There are two core selection policies to choose from (without implementing your own)
+//! `DefaultCoreSelector` and `LookaheadCoreSelector`.
+//!
+//! - The `DefaultCoreSelector` implements a round-robin selection on the cores that can be
+//! occupied by the parachain at the very next relay parent. This is the equivalent to what all
+//! parachains on production networks have been using so far.
```

Comment:
Hmm. Shall we rename this as part of this PR? It seems like LookaheadCoreSelector should be the "default" as we expect any new parachain to use asynchronous backing?